ci/init.js

```diff
@@ -33,6 +33,7 @@ so dd-trace will not be initialized.`)
 
 if (shouldInit) {
   tracer.init(options)
+  tracer.use('fs', false)
```

Comment:
Too many traces generates in jest tests related with fs. This was deactivated also with the old fs plugin.